### ISSUE TITLE
[$20]Adjust Headings on Challenge Details Page

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3248]
**Describe the bug**

Adjust Headings on Challenge Details Page

**To Reproduce**
"1. Go to the Sample challenge page: https://www.topcoder.com/challenges/30095347
2. Check Heading Structure"

**Expected behavior**
(revised)
H1: Challenge Title
H2: "Key Information" instead of Prizes
H2: All Headings in ""Detail"" Tab"

**Actual behavior**
(revised)
Prize as Heading 2.  It should not be H2

**Additional context**

Type: Headings

Page: https://www.topcoder.com/challenges/30098244

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3248 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3248 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3248 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3248 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099941 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099941 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099941 has been updated - it has been assigned to rashmi73.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by rashmi73** [comment]
@veshu 
1.can you please clarify on  ""H1: ""Challenge Details"  i do not find it on page.

**Comment by veshu** [comment]
@crazyk07 @Oanh-and-only-Oanh Can you please help on this?

**Comment by rashmi73** [comment]
@veshu also it would be helpful If I get some inputs on this too " H3: All Headings in ""Detail"" Tab""

**Comment by veshu** [comment]
@rashmi73 This must mean heading inside details tab https://take.ms/5MAgN

**Comment by crazyk07** [comment]
@rashmi73 @veshu  sorry, correction and clarification from Original guidance: 

H1: The Name of the challenge.  This should be H1 on all of our challenge pages.  (Pre-existing)
H2: Veshu is correct with his explanation, except the headings in the details portion are all H2 Headings (Pre-existing)

We need to rename the "Prize" Title to "Key Information" 
![image](https://user-images.githubusercontent.com/43656195/63472917-7c05c680-c428-11e9-999e-6b82965057dc.png)

And be sure it remains a H2 Heading

**Comment by rashmi73** [comment]
@veshu I am completely confused after @crazyk07 commented above.
I feel contradictions in expections still I have raised PR to my best given the requirement of issue stated.

PR https://github.com/topcoder-platform/community-app/pull/3258

**Comment by crazyk07** [comment]
Sorry about that.  Which point should I clarify?

**Comment by rashmi73** [comment]
**H1: The Name of the challenge. This should be H1 on all of our challenge pages. (Pre-existing)**

it is already h1 on challenge page, but issue description says convert it to h2.
so I need to convert it correct?

**Comment by crazyk07** [comment]
Ignore the issue description, it is incorrect.  I edited it originally.

You do not need to change it if it is already there on the page.

**Comment by crazyk07** [comment]
In summary, you only need to change "Prize" to "Key Information" and ensure it remains a H2 Tag.  More simplified than before

**Comment by rashmi73** [comment]
@crazyk07 PR updated according to above comment.
@veshu https://github.com/topcoder-platform/community-app/pull/3258

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099941 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by lakshmiathreya** [comment]
Verified with Chrome on Win10 and MacOS :-
--'Prize' is changed to 'Key Information' and is h2. 
-- Under Details - Heading is h2.
-- Challenge title is h1.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099941<br/><br/>```This is an automated message for crazyk via Topcoder X```
